src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -2045,6 +2059,273 @@ def _pp_asym_gens_output_3ph(self):
         assert "res_asymmetric_sgen_3ph" not in self.pp_output_data
         self.pp_output_data["res_asymmetric_sgen_3ph"] = pp_output_asym_gens_3ph
 
+    # pylint: disable-msg=too-many-locals
+    def _pp_update_loads(self):  # pragma: no cover
+        pp_upd_data = self.pp_update_data["controller"]["object"]
+
+        scaling = self._get_pp_attr("load", "scaling", 1.0)
+        all_load_ids = self.pp_update_data["load"].index.values
+        const_i_multiplier = self._get_pp_attr("load", "const_i_percent", 0) * scaling * (1e-2 * 1e6)
+        const_z_multiplier = self._get_pp_attr("load", "const_z_percent", 0) * scaling * (1e-2 * 1e6)
+        const_p_multiplier = (1e6 - const_i_multiplier - const_z_multiplier) * scaling
+
+        load_controller_ids = []
+        pp_load_ids = set()
+        # Loop over all controllers
+        for count, control in enumerate(pp_upd_data):
+            # If the element of a controller is a load, we save the controller id and load id
+            if control.element == "load":
+                load_controller_ids.append(count)
+                pp_load_ids.add(pp_upd_data[count].element_index[0])
+        # If there are no controllers for loads, we stop here
+        if len(load_controller_ids) < 1:
+            return  # Whether to crash or not
+
+        # Convert it to a list, debugger was complaining
+        pp_load_ids = list(pp_load_ids)
```

Comment:
why does it need to be a `set` in the first place? why not start with a `list`? or, if it should be a `set`, there's probably a problem somewhere down the line 😄

For instance, i don't know if `_get_timeseries_load_ids` wants a `list` as input. If so, either start with a `list` or convert to `list` there.

If you really need the uniqueness feature of `set`, as well as use the `list` output in multiple places, then please update the comment instead